tests/plugins/test_accounts.py

```diff
@@ -252,7 +253,52 @@ def test_permissions_endpoint_is_compatible_with_accounts_plugin(self):
         assert uris == ['/buckets/a/collections/b/records/c',
                         '/buckets/a/collections/b',
                         '/buckets/a',
-                        '/accounts/alice']
+                        '/buckets',
+                        '/accounts/alice',
+                        '/accounts']
+
+    def test_account_create_read_write_permissions_(self):
+        resp = self.app.get('/permissions', headers=self.headers)
+        buckets = resp.json['data']
+        accounts = buckets[4]
+        account = buckets[5]
+        self.assertEqual(len(accounts), 5)
```

Comment:
Here you test the number of keys, I think it would be just easier to compare the dict directly:
```py
self.assertEqual(account, {
    'permissions': ['read', 'write'],
    'resource_name', ...
    ...
})
```